### PR TITLE
Update the watch page post-conference to allow people to re-watch talks

### PR DESCRIPTION
**Pull request by thibaultamartin**

### CODE REVIEW

File: templates/watch.html [hunk 1]
```diff
@@ -1,39 +1,108 @@
...
+            });
+        }
+    });
+</script>
+{% endblock %}
+{% block content %}
+<div id="watch-content" class="page-content">
+    <div id="pills-row">
+        <button class="pill track matrix">Matrix</button>
+        <button class="pill track matrix-in-practice">Matrix in Patrice</button>
```
Comment (HarHarLinks) [inline templates/watch.html:62]:
```suggestion
        <button class="pill track matrix-in-practice">Matrix in Practice</button>
```

**Comment by HarHarLinks** [comment]
We should also update the landing page to be more prominently "conference is over, thanks for attending, watch the recordings and sponsor next year!", underlined by a photo or two. maybe the outside group photo, or maybe from a really crowded talk (~photo policy, ~to be fair to the speaker they should be staff or not clearly recognizable).

**Comment by HarHarLinks** [comment]
I would prefer a lot if the PDFs were committed here rather than linking to google. I can promise you there will be annoyed comments by the community about it.

I hope the images loading ISDN style are just due to pages.dev not being too beefy.

**Comment by MTRNord** [comment]
> I hope the images loading ISDN style are just due to pages.dev not being too beefy.

They are instant on my side. We could test `[loading](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#loading)` however to be lazy to reduce loading at once.

**Comment by MTRNord** [comment]
> I would prefer a lot if the PDFs were committed here rather than linking to google. I can promise you there will be annoyed comments by the community about it.
> 
> -> not a blocker, but preferable and not a lot of work
> 
> I hope the images loading ISDN style are just due to pages.dev not being too beefy.

done the pdf stuff

**Comment by MTRNord** [comment]
> We should also update the landing page to be more prominently "conference is over, thanks for attending, watch the recordings and sponsor next year!", underlined by a photo or two. maybe the outside group photo, or maybe from a really crowded talk (~photo policy, ~to be fair to the speaker they should be staff or not clearly recognizable).
> 
> Thib said:
> 
> > We can make it a big in your face paragraph with 2 CTAs: a secondary CTA 'Sponsor 2025', and a primary CTA 'Watch the talks'
> 
> -> release blockers, imo

Please review text:

grafik


@HarHarLinks @joshsimmons

**Comment by HarHarLinks** [comment]
thanks a lot, MTRNord!

Remaining issues I can see:
- slight layout bug in the rows that have so long speaker names that the speaker names wrap -> not a biggie
- talks without a video link to `#` -> we are expecting to not have a video for everything. we thus need a solution to tell readers "sorry, no video for this (yet)". or at least it should not be clickable.

**Comment by MTRNord** [comment]
> thanks a lot, MTRNord!
> 
> Remaining issues I can see:
> 
>     * slight layout bug in the rows that have so long speaker names that the speaker names wrap -> not a biggie

We noticed this in the last office hour. Sadly we did not get this solved with css.

> 
>     * talks without a video link to `#` -> we are expecting to not have a video for everything. we thus need a solution to tell readers "sorry, no video for this (yet)". or at least it should not be clickable.

This has been fixed. They now are not clickable and there is a pill indicating there is no recording

**Comment by HarHarLinks** [comment]
Nice.

I noticed that for the slides buttons, only the zone marked blue here is actually clickable while the whole area highlights on hover as if it were completely clickable. Again, not a dealbreaker. 
image

**Comment by MTRNord** [comment]
> Nice.
> 
> I noticed that for the slides buttons, only the zone marked blue here is actually clickable while the whole area highlights on hover as if it were completely clickable. Again, not a dealbreaker. image
> 

That should now be fixed :)


>Really nitpicking: For tidiness I would have liked if all pdfs follow the same naming scheme :P

(I just downloaded the thing you gave me and used those :P)

**Pull request merged**